README.md

```diff
@@ -23,15 +23,71 @@ Mental health problems are something that everyone struggles with at various poi
 
 MindGPT sets out to increase ease of access to this information.
 
-# &#128064; Follow along!
+# &#128064; Follow along
 
 This project is in active development at Fuzzy Labs and you can follow along!
 
 The repository for this project is one method where you can monitor progress - we're always raising pull requests to move the project forward. Another way you can do this is to follow our blog. We'll be posting periodic updates as we tick off sections of the project. If you're new to the project, the best place to start is our [project introduction](https://www.fuzzylabs.ai/blog-post/mindgpt-an-introduction).
 
 # &#127939; How do I get started?
 
-We're working on this! Coming soon...
+## Deployment pipeline
+
+To run the zenml deployment pipeline, the resources required by seldon have to provisioned. `matcha` tool can help you in provisioning these resources.
+
+Provision resources using `matcha`
+
+```bash
+matcha provision
+```
+
+Assign output for resources provisioned by `matcha` to variables.
+
+```bash
+function get_state_value() {
+    resource_name=$1
+    property=$2
+    json_string=$(matcha get "$resource_name" "$property" --output json --show-sensitive)
+    value=$(echo "$json_string" | sed -n 's/.*"'$property'": "\(.*\)".*/\1/p')
+    echo "$value"
+}
+
+zenml_storage_path=$(get_state_value pipeline storage-path)
+zenml_connection_string=$(get_state_value pipeline connection-string)
+k8s_context=$(get_state_value orchestrator k8s-context)
+acr_registry_uri=$(get_state_value container-registry registry-url)
+acr_registry_name=$(get_state_value container-registry registry-name)
+seldon_workload_namespace=$(get_state_value model-deployer workloads-namespace)
+seldon_ingress_host=$(get_state_value model-deployer base-url)
+```
+
+Setup zenml stack for deployment
+
+```bash
```

Comment:
Same with this - keeps it a bit tidier (should have said on the call, apologies)